CRM/Speakcivi/Page/Speakcivi.php

```diff
@@ -152,19 +152,22 @@ public function petition($param) {
       $this->setContactCreatedDate($contact['id'], $param->create_dt);
     }
 
+    $opt_in_for_activity_status = $this->opt_in;
```

Comment:
Don't get why you use a temp variable insead of opt_in attribute here.
Beside, the convention for variable names is camelCase, not lowercase_separated_by_underscore, can you change (for this one, and slowly adapt in the future?)

I personally don't have strong preferences, but it's easier for everyone if we stick to the conventions adopted by civicrm core